### PR TITLE
Image bump

### PR DESCRIPTION
**Pull request by krzyzacy**
cc @yguo0905 I bumped the tag but not regenerate your jobs since you'll update them anyway.

include https://github.com/kubernetes/test-infra/pull/3610

/assign @fejta @wojtek-t

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -553,7 +553,7 @@ presubmits:
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
-        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170718-c3574ec6
+        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170720-69a2043d
```
Comment (wojtek-t) [inline prow/config.yaml]: I don't know the answer here, so I'm asking:
is it expected that we have exactly the same hash for both prow and kubeadm images? Why is that?

Comment (krzyzacy) [inline prow/config.yaml]: Since both of them are using https://github.com/kubernetes/test-infra/pull/3611/commits/69a2043d46ec84208edb2432ea7fc88ce10328de

Comment (wojtek-t) [inline prow/config.yaml]: I guess I don't really understand it, but if that's expected I'm fine with that.

Comment (krzyzacy) [inline prow/config.yaml]: @wojtek-t Both of them are based off kubekins-e2e image. The image tag is made from a test-infra commit, https://github.com/kubernetes/test-infra/commit/69a2043d46ec84208edb2432ea7fc88ce10328de, which I bumped both e2e-prow and kubeadm base image tag. And when I push both image they are at the same commit.

**Comment by krzyzacy** [comment]
will mornitor.

also cc @ixdy who's oncall.

File: prow/config.yaml (hunk 1 above)
Comment (wojtek-t) [inline prow/config.yaml]: Aahh ok - now I god it. Thanks!

**Pull request merged**